spec/controllers/transcriptions_controller_spec.rb

```diff
@@ -167,45 +167,65 @@
     end
 
     describe 'roles' do
-      before do
-        get :show, params: { id: transcription.id }
+      shared_examples 'lockable transcription' do
+        let(:locked_transcription) { create(:transcription, locked_by: 'kar-aniyuki', lock_timeout: (DateTime.now + 1.hours)) }
+
+        it 'locks the transcription if not already locked' do
+          get :show, params: { id: transcription.id }
+          expect(json_data['attributes']['locked_by']).to eq(user.login)
+        end
+
+        it 'does not lock the transcription if it is already locked' do
+          get :show, params: { id: locked_transcription.id }
+          expect(json_data['attributes']['locked_by']).to eq(locked_transcription.locked_by)
```

Comment:
Not blocking, you can run these with expect to change syntax as well, https://relishapp.com/rspec/rspec-expectations/v/3-9/docs/built-in-matchers/change-matcher#expect-no-change
```suggestion
          expect { get :show, params: { id: locked_transcription.id } }.not_to change { locked_transcription.locked_by }
```

Reply:
Aha, I like it.